lib/http/request.js

```diff
@@ -126,7 +127,6 @@ class HttpRequest extends EventEmitter {
     this.reqOptions = this.createHttpOptions(options);
     this.hostname = Formatter.formatHostname(this.reqOptions.host, this.reqOptions.port, this.use_ssl);
     this.retryAttempts = this.httpOpts.retry_attempts;
-    this.redact = options.redact || false;
```

Comment:
Remove all occurrences of `this.redact` from the file.

Reply:
done